packages/autorest.gotest/src/generator/goLinter.ts

```diff
@@ -3,20 +3,22 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as path from 'path';
 import { AutorestExtensionHost } from '@autorest/extension-base';
-import { Config, configDefaults } from '../common/constant';
-import { Helper } from '@autorest/testmodeler/dist/src/util/helper';
 import { TestConfig } from '@autorest/testmodeler/dist/src/common/testConfig';
+import { Helper } from '@autorest/testmodeler/dist/src/util/helper';
```

Comment:
Seems we don't have lint rule enabled in this project?

Reply:
what do you mean lint rule?